src/sheet_metal_conus.py

```diff
@@ -261,27 +384,37 @@ def beVerbose(self, dictCone, unitFactor, parent):
                      'y': str(-15) }
         text = inkex.etree.SubElement(parent, 'text', text_atts)
         text.text = "%4.3f" %(longradius)
-        # label for angle
+        # Draw angle
+        lowside = math.degrees(angle) < 180
+        value = math.degrees(angle) if lowside else 360-math.degrees(angle)
+        # radial limit lines
+        line_attribs = {'style' : simplestyle.formatStyle(line_style), 'd' : 'M 3,0 L %4.2f,0' % (ptA[0]*unitFactor*0.8)}
+        line = inkex.etree.SubElement(parent, inkex.addNS('path','svg'), line_attribs)
+        line_attribs = {'style' : simplestyle.formatStyle(line_style), 'd' : 'M %4.2f,%4.2f L %4.2f,%4.2f' % (ptD[0]*unitFactor*0.02, ptD[1]*unitFactor*0.02,ptD[0]*unitFactor*0.8, ptD[1]*unitFactor*0.8)}
+        line = inkex.etree.SubElement(parent, inkex.addNS('path','svg'), line_attribs)
+        # arc
+        arc_rad = ptA[0]*unitFactor*0.50
+        gap = self.getUnittouu(str(font_height*2)+"pt")
+        textpos = self.drawDimArc((0,0), 0, value, arc_rad, arrow_style, parent, gap, lowside)
+        # angle label
+        textpos[1] += font_height/4 if lowside else font_height/2
```

Comment:
Draw the radial limit lines, the arc and the text.
drawDimArc returns the location where text should go. In gap (or middle of line)